gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStatsHttpMonitor.java

```diff
@@ -125,9 +125,8 @@ private OkHttpClient acquireClientWithCookie(String loginUrl)
             return client;
         }
         catch (IOException e) {
-            e.printStackTrace();
+            throw new UncheckedIOException(e);
         }
-        return null;
```

Comment:
This change from using null to throwing exceptions for error handling.
The null checks need to be updated accordingly.
https://github.com/trinodb/trino-gateway/blob/4c40750eb6a1d6710526cdca5d72b2ed139d2740/gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStatsHttpMonitor.java#L137-L140